AdministratorAdministration.yaml

```diff
@@ -1150,8 +1150,8 @@ paths:
                       protocol: 'HTTPS'
                       address:
                         ip-address:
-                          ipv-4-address: '1.2.3.9'
-                      port: 3209
+                          ipv-4-address: '1.2.3.2'
```

Comment:
Since we do not have a live example for https instance of TypeApprovalRegister v1.0.0, can we update this example of TAR to v2.0.0. @openBackhaul Kindly Please let me know your views.